### ISSUE TITLE
Error identifier expected HELP

### ISSUE THREAD

**Issue by ghost** [stanhebben/MineTweaker3#394]
Everytime i reload my `scripts` i get this errer: 
ERROR:Error parsing CustomRecipes.zs:3 -- identifier expected

not sure what it means, but this is the script i wrote for a recipe

```
recipes.addShaped(<LogisticsPipes:item.PipeItemsBasicLogistics>*8,
	[[StructuralPipeFrame, MicroServo, StructuralPipeFrame],
	 [MicroPackager, RoutingLogicProcessor, MicroCapsulator],
	 [StructuralPipeFrame, MicroServo, StructuralPipeFrame]]);
```

and i have a few variables

```
var StructuralPipeFrame = <LogisticsPipes:item.pipeComponents:0>;
var MicroServo = <LogisticsPipes:item.pipeComponents:1>;
var MicroPackager = <LogisticsPipes:item.pipeComponents:3>;
var RoutingLogicProcessor = <LogisticsPipes:item.pipeComponents:2>;
var MicroCapsulator = <LogisticsPipes:item.pipeComponents:04>;
```

can anyone help me?

EDIT: I've fixed it now

**Comment by jaredlll08** [comment]
Upload script to gist please

**Comment by ghost** [comment]
gist?

**Comment by jaredlll08** [comment]
Github gist

**Comment by ghost** [comment]
https://gist.github.com/coopj121/3cf2991e21e89753a8b12eb46c717470

**Comment by ghost** [comment]
i'm new to minetweaker, actually started today so i don't know allot about it

**Comment by jaredlll08** [comment]
Did you get the item names from /mt hand?

**Comment by ghost** [comment]
no i have the item names as a tooltip whenever i hover over the item

**Comment by jaredlll08** [comment]
Please use /mt hand names

**Comment by ghost** [comment]
okay i'll try it now

**Comment by ghost** [comment]
https://gist.github.com/coopj121/3cf2991e21e89753a8b12eb46c717470

still doesn't work, comes out with the same error as before

**Comment by ghost** [comment]
could it be that because i'm using items that don't have crafting recipes yet? i still need to add recipes for them items, i'm planning on using the assembly table to craft the components needed for the basic logistics pipes

**Comment by jaredlll08** [comment]
No, shouldn't be that, try using the actual item names instead of assigning
them to a var?

**Comment by ghost** [comment]
i have it set as the item names now, and it still comes up with the same error?

**Comment by ghost** [comment]
originally what i wanted was the same crafting grid pattern but using the advanced crafting table from buildcraft but i don't think mt has support for that recipe type yet
